activesupport/test/cache/behaviors/cache_store_behavior.rb

```diff
@@ -742,6 +742,11 @@ def test_setting_options_in_fetch_block_does_not_change_cache_options
     end
   end
 
+  def test_configuring_store_with_raw
+    cache = lookup_store(raw: true)
+    assert_equal("bar", cache.fetch("foo") { "bar" })
+  end
```

Comment:
Not sure this is a good enough test.

Perhaps something like:

```ruby
cache.increment("foo")
assert_equal "1", cache.read("foo")
```
? This way we test the read was implictly raw.